powershellgroup/powershellgroup.resource.ps1

```diff
@@ -299,22 +299,26 @@ elseif ($Operation -eq 'Test')
         if ($cachedResourceInfo)
         {
             $inputht = @{}
-            $ResourceTypeName = ($inputobj_pscustomobj.type -split "/")[1]
-            $inputobj_pscustomobj.psobject.properties | %{ 
-                if ($_.Name -ne "type")
-                {
-                    $inputht[$_.Name] = $_.Value
+            $ResourceTypeName = ($r.type -split "/")[1]
```

Comment:
If I'm reading this right, here (around line 302) `$r` is not defined.
Looks like you wanted to use `$inputobj_pscustomobj` instead of `$r`.